runtime/lua/vim/lsp/handlers.lua

```diff
@@ -462,6 +462,8 @@ M[ms.textDocument_typeDefinition] = location_handler
 --- @see # https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_implementation
 M[ms.textDocument_implementation] = location_handler
 
+local sig_help_ns = api.nvim_create_namespace('lsp_signature_help')
```

Comment:
nit/non-blocker: do we usually prefix our namespaces with `nvim_` ? 

also wondering if we should use dot "." as the separator for namesapces, e.g. `nvim.lsp.foo`.

Reply:
I don't think we have a convention regarding namespace names. E.g. we use [`'vim/snippet'`](https://github.com/neovim/neovim/blob/36890824c7edb7bc5458062c14efbfa0b1d280b5/runtime/lua/vim/snippet.lua#L3) in `vim.snippet`, [`'ts.dev'`](https://github.com/neovim/neovim/blob/36890824c7edb7bc5458062c14efbfa0b1d280b5/runtime/lua/vim/treesitter/dev.lua#L138) in the TS inspector, etc.

We do seem to prefix these with `vim_` in the LSP namespaces though, so I'll stick to that.